test_conformance/non_uniform_work_group/TestNonUniformWorkGroup.cpp

```diff
@@ -706,12 +706,38 @@ size_t TestNonUniformWorkGroup::adjustGlobalBufferSize(size_t globalBufferSize)
   size_t adjustedGlobalBufferSize = globalBufferSize;
   if (deviceMaxAllocObjSize < globalBufferSize) {
     adjustedGlobalBufferSize = deviceMaxAllocObjSize;
-    log_info("globalBufferSize was adjusted from %lu to %lu\n", globalBufferSize, adjustedGlobalBufferSize);
   }
 
   return adjustedGlobalBufferSize;
 }
 
+void TestNonUniformWorkGroup::adjustGlobalSize(size_t maxNumOfGlobalWorkItems)
```

Comment:
I am thinking if better solution to have one common function with `adjustGlobalBufferSize`

Reply:
merged into adjustGlobalBufferSize